app/models/user.rb

```diff
@@ -2,14 +2,14 @@ class User < ActiveRecord::Base
   include PgSearchCommon
 
   has_attached_file :avatar,
-    styles: { medium: "300x300>", thumb: "100x100>#", head: "32x32#" },
+    styles: { medium: "300x300>", thumb: "100x100>", head: "32x32#" },
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.